coremltools/models/neural_network/builder.py

```diff
@@ -723,8 +737,8 @@ def _check_fp16_weight_params_lstms(self, lstm_wp, has_peephole=True):
 
         Parameters
         ----------
-        lstm_wp: lstm weights
-        has_peephole: if the lstm has peephole
+        lstm_wp: ``lstm`` weights.
```

Comment:
It doesn't look like "lstm" refers to a variable. So I probably wouldn't worry wouldn't add double backticks.

In general, I don't think you need to worry too much about docstrings in what are called "hidden" methods (i.e. methods whose name starts with an underscore). Those docstring should never show up in API docs.